eng/packages/http-client-csharp/generator/Azure.Generator/src/ConfigurationExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.TypeSpec.Generator;
+
+namespace Azure.Generator
+{
+    internal static class ConfigurationExtensions
+    {
+        public static bool UseModelNamespace(this Configuration config) =>
```

Comment:
This may be incompatible with existing conventions, but I was just curious - would there be any value in differentiating between explicit true/false and "unset" via nullability?

Reply:
The usage of this method is internal to the generator, so I don't think it would make a difference. For mgmt to change the default, they would essentially check whether the user has set this option, and if not, pass true in the emitter code.